### PR TITLE
Grid: Two types of grids containers are added. 

### PR DESCRIPTION
**Pull request by vaishnavipaliya11**
Grid with two elements in it. 
grid with three elements in it.
image
image

### CODE REVIEW

File: grids/grids.html [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+            style="
+              width: 682px;
+              height: 654px;
+              border: 0;
+              transform: scale(1);
+              overflow: hidden;
+            "
```
Comment (devsurajrai) [inline grids/grids.html]: Please avoid inline styling.

**Review by devsurajrai — COMMENTED** [review]
Code is well written. Good Job . Just as the suggestion says try to avoid any inline styling in your code.

File: grids/grids.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
+.grid-two-element-container {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-template-areas: "one two";
```
Comment (saifudinbadani) [inline grids/grids.css:4]: Good use of templates, it would be good if you can use meaningful names.

**Review by saifudinbadani — APPROVED** [review]
Other than the comment, rest of the code looks good.

**Review by yogesh- — COMMENTED** [review]
Looks good.

File: grids/grids.css (hunk 2 above)
Comment (vaishnavipaliya11) [reply grids/grids.css:4]: okay

**Comment by sushmita12345** [comment]
- Proper naming for CSS class.
- Code is proper structured.
- Mostly you have follow the good practices for responsiveness.
- Code looks good, you can proceed to merge.

**Review by sushmita12345 — APPROVED** [review]

**Pull request merged**